python_modules/dagster/dagster/_core/definitions/multi_dimensional_partitions.py

```diff
@@ -0,0 +1,76 @@
+from typing import List, Mapping, NamedTuple, Optional, Sequence, Union
+
+import dagster._check as check
+from dagster._serdes import whitelist_for_serdes
+from dagster._serdes.serdes import deserialize_as, serialize_dagster_namedtuple
+
+
+@whitelist_for_serdes
+class PartitionDimensionKey(
+    NamedTuple(
+        "_PartitionDimensionKey",
+        [
+            ("dimension_name", str),
+            ("partition_key", str),
+        ],
```

Comment:
If you get rid of trailing commas, black might be able to format this more concisely.